### ISSUE TITLE
Data preprocess

### ISSUE THREAD

**Issue by xh-xh001** [facebookresearch/banmo#8]
#Thanks for your excellent work!
##An error is displayed during data preprocessing.The environment configuration is configured according to the “banmo.yml” you provided. Tested with video of my own data, but could not preprocess. Using the series of videos(“cat-pikachiu.mov”) to process, it still reports an error.
the backtrack of error logger is:
bash preprocess/preprocess.sh ”cat-pikachiu“ .MOV n 10 
<img width="674" alt="image" src="https://user-images.githubusercontent.com/70375440/158557713-7a629ec6-7ab2-4e8d-baf9-6bdeacb04f50.png">

**Comment by xh-xh001** [comment]
> #Thanks for your excellent work! ##An error is displayed during data preprocessing.The environment configuration is configured according to the “banmo.yml” you provided. Tested with video of my own data, but could not preprocess. Using the series of videos(“cat-pikachiu.mov”) to process, it still reports an error. the backtrack of error logger is: bash preprocess/preprocess.sh ”cat-pikachiu“ .MOV n 10 <img alt="image" width="674" src="https://user-images.githubusercontent.com/70375440/158557713-7a629ec6-7ab2-4e8d-baf9-6bdeacb04f50.png">

**Comment by minushuang** [comment]
> #Thanks for your excellent work! ##An error is displayed during data preprocessing.The environment configuration is configured according to the “banmo.yml” you provided. Tested with video of my own data, but could not preprocess. Using the series of videos(“cat-pikachiu.mov”) to process, it still reports an error. the backtrack of error logger is: bash preprocess/preprocess.sh ”cat-pikachiu“ .MOV n 10 <img alt="image" width="674" src="https://user-images.githubusercontent.com/70375440/158557713-7a629ec6-7ab2-4e8d-baf9-6bdeacb04f50.png">

hi, same with you , did you solve the problem?

**Comment by gengshan-y** [comment]
Is this the first error that showed up? Is the [data](https://github.com/facebookresearch/banmo#data) downloaded?

**Comment by minushuang** [comment]
> Is this the first error that showed up? Is the [data](https://github.com/facebookresearch/banmo#data) downloaded?

the data is my own video with a cat, and the total images number is 93 in `database/DAVIS/JPEGImages/Full-Resolution/$seqname/*.jpg` extracted from the cat video. 
and the problem occurred  when the` jnx >= 93` in the while loop in `main()` funciton in `auto_gen.py`.
my solution is check the jnx and break when` jnx >= 93` in the while loop
not robust method. I think there must be something I failed to notice.  
do you have any suggestion?

**Comment by gengshan-y** [comment]
It's safe to ignore this out of range error in auto_gen.py. Indeed it should work even with this error.  There was supposed to be a break [here](https://github.com/facebookresearch/banmo/blob/7773a11594fb92866658747f0e59e15906a54a1d/third_party/vcnplus/auto_gen.py#L155)  if `jnx>=len(test_left_image)`.

Besides that, are there other issues?

**Comment by minushuang** [comment]
no, the optimization process is running normally at current time.
